### ISSUE TITLE
Unable to parse PDF from existing PDF

### ISSUE THREAD

**Issue by garrettApproachableGeek** [DavBfr/dart_pdf#1192]
**Describe the bug**

When using the following code on web and iOS, we are unable to load a particular pdf document
Using: 
`pdf_crypto: ^3.2.7`
`pdf: ^3.7.4`

**To Reproduce**
Code snippet to reproduce the behavior:
```
pdf_crypto.PdfDocumentParser(bytes),
```

**Expected behavior**

The PDF document is parsed and returns a `PdfDocumentParserBase`

**Flutter Doctor**

```
[✓] Flutter (Channel stable, 2.10.2, on macOS 12.6 21G115 darwin-arm, locale en-US)
    • Flutter version 2.10.2 at /Users/garrett/fvm/versions/2.10.2
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 097d3313d8 (9 months ago), 2022-02-18 19:33:08 -0600
    • Engine revision a83ed0e5e3
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at /Users/garrett/Library/Android/sdk
    • Platform android-33, build-tools 32.1.0-rc1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[✓] VS Code (version 1.73.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.52.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 12.6 21G115 darwin-arm
    • Chrome (web)    • chrome • web-javascript • Google Chrome 107.0.5304.87

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Desktop (please complete the following information):**
 - [X] iOS
 - [ ] Android
 - [X] Browser
 - [ ] Windows
 - [ ] Linux

**Chrome:**
 - Device: [Macbook Pro M1 Pro]
 - OS: [MacOS]
 - Browser [Chrome]
 - Version [Version 107.0.5304.87 (Official Build) (arm64)]

**iOS:**
 - Device: [iPad 9th gen]
 - OS: [iOS]
 - Browser [N/A]
 - Version [15.3.1]

**Additional context**

[pdf_with_issue.pdf](https://github.com/DavBfr/dart_pdf/files/9962601/A-7.1.pdf)

It seems this error is thrown in a uft8 decode within `_loadPageLabels` of `document.dart`
```
Unhandled Exception: FormatException: Invalid UTF-8 byte (at offset 0)
#0      _Utf8Decoder.convertSingle (dart:convert-patch/convert_patch.dart:1789:7)
#1      Utf8Decoder.convert (dart:convert/utf.dart:351:42)
#2      Utf8Codec.decode (dart:convert/utf.dart:63:20)
#3      PdfDocumentParser._loadPageLabels (package:pdf_crypto/src/parser/document.dart:189:20)
#4      new PdfDocumentParser (package:pdf_crypto/src/parser/document.dart:74:7)
```

**Comment by DavBfr** [comment]
Should be fixed in pdf_crypto 3.2.8.
https://pub.nfet.net/pdf_crypto/

**Comment by garrettApproachableGeek** [comment]
Thanks, I have confirmed this fixes the issue
